crates/orderbook/src/database/orders.rs

```diff
@@ -305,6 +305,17 @@ impl OrderStoring for Postgres {
                     if let Some(quote) = new_quote {
                         insert_quote(&new_order.metadata.uid, &quote, ex).await?;
                     }
+                    if let Some(full_app_data) = new_order.metadata.full_app_data {
```

Comment:
Isn't `fullAppData` mandatory for the replace order?

Reply:
@alfetopito it is indeed, but it will always be `Some(_)` here. In order to change this I would need to create new structs and change the signature of a couple of functions.